docs/faq.md

```diff
@@ -234,9 +234,9 @@ It's the same with email servers. Your email address is likely `name@company.com
 
 Using a separate domain name is easier to manage (although it's a little hard to get right at first) and keeps your Matrix server isolated from your website (if you have one), from your email server (if you have one), etc.
 
-We allow `matrix.example.com` to be the Matrix server handling Matrix stuff for `example.com` by [Server Delegation](howto-server-delegation.md). During the installation procedure, we recommend that you set up server delegation using the [.well-known](configuring-well-known.md) method.
+By default, this playbook sets up services on your Matrix server (`matrix.example.com`). To have this server officially be responsible for Matrix services for the base domain (`example.com`), you need to set up [Server Delegation](howto-server-delegation.md). This is normally done by [configuring well-known](configuring-well-known.md) files on the base domain.
 
-If you'd really like to install Matrix services directly on the base domain, see [How do I install on matrix.example.com without involving the base domain?](#how-do-i-install-on-matrixexamplecom-without-involving-the-base-domain)
+If you'd really like to install Matrix services directly on the base domain (or need to do so), see [How do I install on matrix.example.com without involving the base domain?](#how-do-i-install-on-matrixexamplecom-without-involving-the-base-domain)
```

Comment:
This points to a FAQ entry that tells you how to setup directly on the matrix domain (so that you don't need to involve the base domain).

It's not for "installing directly on the base domain". While the latter can be achieved by tweaking settings, the FAQ entry is not for it.